File: VideoApp/app/src/main/java/learn/exercicio/videoapp/MainActivity.java
package learn.exercicio.videoapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void clickEquipe(View view) {
        Intent intent = new Intent(this, Equipe.class);
        startActivity(intent);
    }

    public void clickCursos(View view) {
        Intent intent = new Intent(this, Cursos.class);
        startActivity(intent);
    }

    public void clickMidia(View view) {
        Intent intent = new Intent(this, Midia.class);
        startActivity(intent);
    }
}

File: ListaFilmes/app/src/main/java/learn/exercicio/listafilmes/MainActivity.kt
package learn.exercicio.listafilmes

import android.graphics.Movie
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    private val listOfMovies = arrayListOf(
        Movie(
            id = 0,
            titulo = "Titanic",
            descricao = null,
            imagem = null,
            dataLancamento = null
        ),
        Movie(
            id = 0,
            titulo = "Central do Brasil",
            descricao = null,
            imagem = null,
            dataLancamento = null
        )
    )

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }

    private fun populateList() {
        moviesList.apply {
            hasFixedSize()
        adapter = MoviesAdapter(listOfMovies)
        }
    }
}
